keps/2724-topology-aware-scheduling/README.md

```diff
@@ -430,9 +430,16 @@ const (
   // at the highest topology level, then it gets admitted as distributed
   // among multiple topology domains.
   PodSetPreferredTopologyAnnotation = "kueue.x-k8s.io/podset-preferred-topology"
+
+  // PodSetUnconstrainedTopologyAnnotation indicates that a PodSet requires no topology.
+  // Kueue admits the PodSet if there's enough free capacity anywhere in the cluster.
+  // Recommended for PodSets that don't require inter pod communication
+  PodSetUnconstrainedTopologyAnnotation = "kueue.x-k8s.io/podset-unconstrained-topology"
 )
 ```
 
+The annotations are mutually exclusive and shouldn't be combined.
```

Comment:
I would suggest to move this note to the Validation section, as we already validate against it.

Reply:
Done